.gitmodules

```diff
@@ -23,7 +23,7 @@
 	url = https://github.com/pulp-platform/common_verification.git
 [submodule "hardware/deps/cva6"]
 	path = hardware/deps/cva6
-	url = https://github.com/pulp-platform/cva6.git
+	url = git@github.com:pulp-platform/cva6.git
```

Comment:
Is there a reason why you changed the `url`?

Reply:
Error: https://github.com/pulp-platform/ara/actions/runs/4065579651/jobs/7000552212#step:9:22

Reply:
I encountered a problem with my fork repository using the SSH URL while trying to bump the repository, but I think it will function properly here. So, I will change it and remove branch as well.